src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -125,32 +125,28 @@ class SearchControls extends Component {
   }
 
   renderSelectOptionWithIcon(facilityType) {
-    switch (facilityType) {
-      case 'health':
-        return (
-          <button type="button" className="facility-option">
-            <span className="flex-center"><span className="legend health-icon"></span>Health</span>
-          </button>
-        );
-      case 'benefits':
-        return (
-          <button type="button" className="facility-option">
-            <span className="flex-center"><span className="legend benefits-icon"></span>Benefits</span>
-          </button>
-        );
-      case 'cemetery':
-        return (
-          <button type="button" className="facility-option">
-            <span className="flex-center"><span className="legend cemetery-icon"></span>Cemetery</span>
-          </button>
-        );
-      default:
-        return (
-          <button type="button" className="facility-option">
-            <span className="flex-center all-facilities"><span className="legend spacer"></span>All Facilities</span>
-          </button>
-        );
+    /* eslint-disable camelcase */
+    const classMappings = {
+      va_health_facility: 'health',
+      va_cemetery: 'cemetery',
+      va_benefits_facility: 'benefits',
+      vet_center: 'vet-center',
+    };
+    /* eslint-enable camelcase */
+
+    if (facilityType) {
+      return (
+        <button type="button" className="facility-option">
+          <span className="flex-center"><span className={`legend ${classMappings[facilityType]}-icon`}></span>{facilityTypes[facilityType]}</span>
```

Comment:
Seems like `classNames` could be used here.

```
const legendClass = classNames(
  'legend',
  `${classMappings[facilityType]}-icon`
);
```

Also could potentially avoid the `eslint-disable` with something like:


```
const legendClass = classNames(
  'legend',
  { 'health-icon': facilityType === 'va_health_facility' },
  { 'cemetary-icon': facilityType === 'va_cemetary' },
  ...
);
```